kt/entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/ext/StringExtensions.kt

```diff
@@ -0,0 +1,7 @@
+package godot.entrygenerator.ext
+
+fun String.camelToSnakeCase(): String {
```

Comment:
note for future, should be merged with string operations we have in api generator.